### PR TITLE
Forbids net connection during tests

### PR DESCRIPTION
**Pull request by braktar**

### CODE REVIEW

**Comment by braktar** [comment]
It prevents any test from calling an external service on the internet

**Comment by senhalil** [comment]
> It prevents any test from calling an external service on the internet

:+1:  Oh, I thought it works only on already mocked or stubbed requests.

**Comment by senhalil** [comment]
Which also means that if we ever want to use a test to quickly launch a new instance or TDD we need to have `WebMock.allow_net_connect!` in the test.

File: test/test_helper.rb [hunk 1]
```diff
@@ -120,7 +123,9 @@ def self.create(problem)
   def self.matrix_required(vrp)
     return unless ENV['TEST_DUMP_VRP'].to_s == 'true' && vrp.name
 
+    WebMock.enable_net_connect!
     vrp.compute_matrix
+    WebMock.disable_net_connect!
```
Comment (braktar) [inline test/test_helper.rb:128]: This should allow us to define new tests and avoid locals `WebMock.enable_net_connect!`

**Review by fonsecadeline — APPROVED** [review]

File: test/test_helper.rb (hunk 1 above)
Comment (senhalil) [reply test/test_helper.rb:128]: You tried to dump a tests and it works right? Do you know by any chance what `allow_net_connection` does by the way?

**Review by senhalil — APPROVED** [review]

**Comment by braktar** [comment]
`allow_net_connect` and `disable_net_connect` are aliases : https://github.com/bblimke/webmock/pull/789/files

**Pull request merged**